### PR TITLE
Minimal demo for ghpages using jupyter lite

### PR DESCRIPTION
**Pull request by gjmooney**
Addresses #2. Provides a minimal demo with Jupyter Lite with an input field that allows sending arbitrary commands from the host page. Includes a `jupyter-lite.json` to showcase disabling extensions to remove UI elements from the base UI.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Core modules (mandatory)
+jupyterlite-core==0.4.4
```
Comment (jtpio) [inline requirements.txt:2]: Maybe we can also add a kernel, for example `jupyterlite-pyodide-kernel==0.4.4`?

File: demo/index.html [hunk 2]
```diff
@@ -27,12 +27,31 @@
...
   </head>
   <body>
-    <h1>Parent Page</h1>
+    <div
+      style="display: flex; align-items: center; justify-content: space-around"
+    >
+      <h1>Demo</h1>
+      <div>
+        <form id="commands">
+          <input type="text" name="command" placeholder="Enter a command" />
```
Comment (jtpio) [inline demo/index.html:36]: Should we also have another input to specify (optional) args?

File: jupyter-lite.json [hunk 3]
```diff
@@ -0,0 +1,11 @@
+{
+  "jupyter-lite-schema-version": 0,
+  "jupyter-config-data": {
+    "disabledExtensions": [
+      "jupyterlab-kernel-spy",
+      "jupyterlab-tour",
```
Comment (jtpio) [inline jupyter-lite.json:6]: Do we need to disabled these extensions? It looks like they are not installed (not listed in the `requirements.txt`).

File: demo/index.html (hunk 2 above)
Comment (gjmooney) [reply demo/index.html:36]: Added!

**Pull request merged**